pkg/controller/podautoscaler/horizontal_test.go

```diff
@@ -154,6 +160,9 @@ func (tc *testCase) prepareTestClient(t *testing.T) *fake.Clientset {
 				},
 			},
 		}
+		tc.Lock()
```

Comment:
This should be moved to the beginning of the function - you're reading values from `tc` when creating `obj`